draft-wuertele-oauth-security-topics-update.md

```diff
@@ -179,21 +179,21 @@ authorization server metadata.
 
 Excerpt from H-AS' metadata:
 
-~~~ javascript
+~~~ json
```

Comment:
Unfortunately, this cannot be `json`, because that leads to errors in the build pipeline (complaining that this is not valid JSON, this is true). We consciously decided not to omit the `...`  to clearly indicate that further metadata is needed.